deploy/kube-app-manager-aio.yaml

```diff
@@ -0,0 +1,780 @@
+# Copyright 2020 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: Namespace
```

Comment:
@barney-s 
Is there a way to easily add e2e on this file ?  
I'm thinking of:
```
#1
./create-kubernetes-cluster-script.sh #or whatever command
#2
kubectl apply -f kube-app-manager-aio.yaml
#3
./run-e2e-tests
```
I think we already have #1 and #3, could you point me where I should plug step #2 ?

Reply:
if you follow the suggestions in makefile. nothing needs to be done here.

Reply:
I'll look at it in a follow-up pr